gradle/libs.versions.toml

```diff
@@ -83,6 +83,17 @@ coroutines = "1.4.2"
 # Local Server
 ktor = "1.6.0"
 
+# Otel
+slf4j = "1.7.30"
```

Comment:
this is just the `API` which under the hood it checks for the implementation and if it can't find any it defaults to the NoOp one. This package is used all over the Java code so it will be huge to start removing all its usage.